src/transforms/filter.jl

```diff
@@ -69,22 +65,45 @@ DropMissing(cols::T...) where {T<:ColSelector} =
 
 isrevertible(::Type{<:DropMissing}) = true
 
-_ftrans(::DropMissing{Colon}, table) =
+# ftrans
+_ftrans(::DropMissing{Colon}, cols) =
   Filter(row -> all(!ismissing, row))
 
-function _ftrans(transform::DropMissing, table)
-  allcols = Tables.columnnames(table)
-  cols = _filter(transform.colspec, allcols)
+_ftrans(::DropMissing, cols) =
   Filter(row -> all(!ismissing, getindex.(Ref(row), cols)))
-end
+
+# nonmissing 
+_nonmissing(::Type{T}, c) where {T} = c
+_nonmissing(::Type{Union{Missing,T}}, c) where {T} =
+  collect(T, c)
+
+_nonmissing(col) = _nonmissing(eltype(col), col)
 
 function apply(transform::DropMissing, table)
-  ftrans = _ftrans(transform, table)
+  colnames = Tables.columnnames(table)
+  select = _filter(transform.colspec, colnames)
+  ftrans = _ftrans(transform, select)
   newtable, fcache = apply(ftrans, table)
-  newtable, (ftrans, fcache)
+
+  # post-processing
+  coltable = Tables.columntable(newtable)
+  pcolumns = [nm ∈ select ? _nonmissing(col) : col for (nm, col) in pairs(coltable)]
```

Comment:
In other transforms we call the vector of values of a column `x`. Can you also use this letter here? Same for the implementations of _nonmissing, they should get a vector x instead of a vector c

Reply:
Done with 7db5e28